engine/src/main/scala/cromwell/engine/workflow/lifecycle/materialization/MaterializeWorkflowDescriptorActor.scala

```diff
@@ -173,15 +155,16 @@ class MaterializeWorkflowDescriptorActor(serviceRegistryActor: ActorRef,
 
   startWith(ReadyToMaterializeState, ())
 
+
   when(ReadyToMaterializeState) {
-    case Event(MaterializeWorkflowDescriptorCommand(workflowSourceFiles, conf), _) =>
-      val replyTo = sender()
 
+    case Event(MaterializeWorkflowDescriptorCommand(workflowSourceFiles, conf, callCachingEnabled, invalidateBadCacheResults), _) =>
+      val replyTo = sender()
```

Comment:
`pipeTo`?

Reply:
Here is a comment: 
Pipe the response to self, but make it look like it comes from the sender of the command . This way we can access it through sender() in the next state and don't have to store the value of replyTo in the data.

pipe(futureDescriptor).to(self, replyTo)
Is that what you meant?